website/blog/2019-06-26-7.5.0.md

```diff
@@ -186,12 +187,13 @@ module.exports = {
 
 If you want to only allow parsing of `import()` expressions without transforming them, you can just include the `@babel/plugin-syntax-dynamic-import` package.
 
-If you are using `@babel/preset-env`, dynamic import support will be enabled by default. You don't need to worry about webpack or Rollup support, since `babel-loader` and `rollup-plugin-babel` will automatically disable the Babel transform to let the bundler handle it correctly.
+If you are using `@babel/preset-env`, dynamic import support will be enabled by default. You don't need to worry about `webpack` or `rollup` support, since `babel-loader` and `rollup-plugin-babel` will automatically disable the Babel transform to let the bundler handle it correctly.
 
 ### `defaults` browserlist query in `@babel/preset-env` ([#8897](https://github.com/babel/babel/pull/8897))
 
 By default `@babel/preset-env` runs every syntax transform on your code, making it compatible with ECMAScript engines. This often leads to unnecessary transforms, not needed for most browsers.
-For this reason, Babel users should have always asked theirselves which browsers they wanted to support, or which usage constraints (e.g. `> 5%, not IE`). Browserslist, the library `@babel/preset-env` uses to decode your targets, can take away this decision by providing [sensisble default choices](https://browserl.ist/?q=defaults). 
+
+For this reason, Babel users should always ask themselves which browsers they want to support or their constraints (e.g. `> 5%, not IE`). Browserslist, the library `@babel/preset-env` uses to decode your targets, can help with this decision by providing [sensisble default choices](https://browserl.ist/?q=defaults). 
```

Comment:
> For this reason, Babel users should always ask themselves which browsers they want to support

I think this sentence should use a past tense, since now users can use `default` and avoid thinking about it.